sources/editor/Xenko.Assets.Presentation/AssetEditors/Gizmos/NavigationBoundingBoxGizmo.cs

```diff
@@ -87,7 +87,7 @@ public void Build()
 
                 vertices[0] = new VertexPositionNormalTexture(new Vector3(-1, 1, -1), Vector3.UnitY, Vector2.Zero);
                 vertices[1] = new VertexPositionNormalTexture(new Vector3(-1, 1, 1), Vector3.UnitY, Vector2.Zero);
-                vertices[2] = new VertexPositionNormalTexture(new Vector3(1, 1, 1), Vector3.UnitY, Vector2.Zero);
+                vertices[2] = new VertexPositionNormalTexture(Vector3.One, Vector3.UnitY, Vector2.Zero);
```

Comment:
I think the idea here was to be symmetric with the above and below lines.
Using `Vector.One` here doesn't provide any performance improvement since it is a `struct`, under the hood it is basically doing the same

Reply:
Good catch, thanks. I did a bulk find and replace and tried to undo the ones that looked more sensible in the old form, and clearly I didn't missed undoing some. I'll fix this, along with any other if you spot them.